src/ibex_bluesky_core/callbacks/_fitting.py

```diff
@@ -251,3 +253,138 @@ def write_fields_table_uncertainty(self) -> None:
 
         rows = zip(self.x_data, self.y_data, self.yerr_data, self.y_fit_data, strict=True)
         self.csvwriter.writerows(rows)
+
+
+class ChainedLiveFit(CallbackBase):
+    """Processes multiple LiveFits, each fit's results inform the next, with optional plotting.
+
+    This callback handles a sequence of LiveFit instances where the parameters from each
+    completed fit serve as the initial guess for the subsequent fit. Optional plotting
+    is built in using LivePlotLib.
```

Comment:
`LivePlotLib` is new to me! (I think you meant either `LivePlot` or `matplotlib` and merged them ;) )